actionmailbox/app/controllers/rails/conductor/action_mailbox/inbound_emails_controller.rb

```diff
@@ -22,7 +22,7 @@ def create
       def new_mail
         Mail.new(mail_params.except(:attachments).to_h).tap do |mail|
           mail[:bcc]&.include_in_headers = true
-          mail_params[:attachments].to_a.each do |attachment|
+          mail_params.param_at(:attachments).to_a.each do |attachment|
```

Comment:
This change shouldn't be necessary but was required to get the test green.
It's probably a bug.